packages/ibm-products/src/components/FullPageError/FullPageError.js

```diff
@@ -0,0 +1,152 @@
+
+      // Collect any other property values passed in.
+      ...rest
+    },
+    ref
+  ) => {
+    return (
+      <div
+        {
+          // Pass through any other property values as HTML attributes.
+          ...rest
+        }
+        className={cx(
+          blockClass, // Apply the block class to the main HTML element
+          className, // Apply any supplied class names to the main HTML element.
+          // example: `${blockClass}__template-string-class-${kind}-n-${size}`,
+          {
+            // switched classes dependant on props or state
+            // example: [`${blockClass}__here-if-small`]: size === 'sm',
+          }
+        )}
+        ref={ref}
+        role="main"
+        {...getDevtoolsProps(componentName)}
+      >
+        <div className={`${blockClass}__error-container`}>
+          <Grid className={`${blockClass}__error-grid`}>
+            <Column
+              sm={4}
+              md={3}
+              lg={6}
+              className={`${blockClass}__error-column`}
+            >
+              {errorLabel && (
+                <p className={`${blockClass}__error-label`}>{errorLabel}</p>
+              )}
+              {title && (
+                <h1 className={`${blockClass}__error-title`}>{title}</h1>
+              )}
+              {description && (
+                <p className={`${blockClass}__description`}>{description}</p>
+              )}
+              {children}
+            </Column>
+            <Column sm={4} md={5} lg={10}>
+              <div
+                style={{
+                  display: 'flex',
+                  height: '100%',
+                }}
+              >
+                <ErrorGenericSVG className={`${blockClass}__error-svg`} />
+              </div>
+            </Column>
+          </Grid>
+        </div>
+      </div>
+    );
+  }
+);
+
+// Return a placeholder if not released and not enabled by feature flag
+FullPageError = pkg.checkComponentEnabled(FullPageError, componentName);
+
+// The display name of the component, used by React. Note that displayName
+// is used in preference to relying on function.name.
+FullPageError.displayName = componentName;
+
+// The types and DocGen commentary for the component props,
+// in alphabetical order (for consistency).
+// See https://www.npmjs.com/package/prop-types#usage.
+FullPageError.propTypes = {
+  /**
+   * Provide the contents of the FullPageError.
+   */
+  children: PropTypes.node,
+
+  /**
+   * Provide an optional class to be applied to the containing node.
+   */
+  className: PropTypes.string,
+  /**
+   * String that will provide the description for the error code
+   */
+  description: PropTypes.string.isRequired,
+  /**
+   * String that will describe the error that occurred
+   */
+  errorLabel: PropTypes.string.isRequired,
+  /**
+   * This will be for the main title of the FullPageError component
+   */
+  title: PropTypes.string,
```

Comment:
Believe `title` should also be required.

```suggestion
  title: PropTypes.string.isRequired,
```